app/helpers/application_helper.rb

```diff
@@ -200,6 +200,10 @@ def url_for_db(db, action = "show", item = nil) # Default action is show
                     )
     elsif @host && ["Patch", "GuestApplication"].include?(db)
       return url_for(:controller => "host", :action => @lastaction, :id => @host, :show => @id)
+    elsif db == "MiqCimInstance" && @db && @db == "snia_local_file_system"
```

Comment:
these 4 lines are obsolete, this functionality got removed in #204 (https://github.com/ManageIQ/manageiq-ui-classic/pull/204/files#diff-127a850b7d7b20173ad3fba88f8a25e3L203)